clients/fides-js/src/components/CloseButton.tsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable react/require-default-props */
```

Comment:
we should just turn this off in eslintrc like we do for admin-ui and privacy-center

https://github.com/ethyca/fides/blob/feature/privacy-components/clients/admin-ui/.eslintrc.json/#L37-L38